app/models/badges/completer_badge.rb

```diff
@@ -0,0 +1,18 @@
+module Badges
+  class CompleterBadge < Badge
+    seed "Completer",
+      :ultimate,
```

Comment:
I made them ultimate. I've not yet done the bastion check to see how many users would get this badge.